### PR TITLE
GitHub Actions CI (ported from TravisCI)

### PR DESCRIPTION
**Pull request by tomkinsc**
This adds a file, `.github/workflows/build.yml`, to port from ​from TravisCI to GitHub Actions the following: WDL validation, integration testing, and DNAnexus workflow deployment.

Integration testing is dependent on validation checks, as before. DNAnexus deployment is contingent on validation by both miniwdl and womtool (since dxWDL tends to be more strict).

DNAnexus build+deployment is set to be conditional on the merge of a PR.

This also updates the DNAnexus runner to py3.8 (now supported by dx-toolkit).

A weekly cron test is added to run checks on Mondays.

The intent here is to run these checks in parallel with the TravisCI checks until we are confident in the equivalence and reliability of GitHub actions, at which time we can disable TravisCI checks and remove them from the `master` branch protection rules.

### CODE REVIEW

**Comment by dpark01** [comment]
this is so exciting! (as technical debt goes…)

**Comment by tomkinsc** [comment]
Builds seem to be faster than Travis!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
+name: "viral-pipelines CI"
+
+on:
+  push:
+    branches:
+      - master
```
[the remark this answers is not in the document]
Comment (tomkinsc) [reply .github/workflows/build.yml:6]: Yeah, let me change that to run on push to any branch.

**Review by dpark01 — APPROVED** [review]
This looks good. I actually imagine that we might cut over sooner than later, but let's see how the coming week goes with one foot in both CIs. Much appreciation for tackling this.

**Pull request merged**